metricbeat/module/uwsgi/status/_meta/fields.yml

```diff
@@ -0,0 +1,121 @@
+- name: status
+  type: group
+  description: >
+    uwsgi.status metricset fields
+  fields:
+    - name: total.requests
```

Comment:
I would probably turn around the namespacing as I think`requests` is the namespace and `total` only describes which values it is in the `requests` namespace. Same below like:


`errors.write.total`
`errors.read.total`

and others.

Reply:
`total` was kind of summary for all workers, it has `requests, exceptions, read, write errors`, if I turnaround total.requests, can you suggest naming for summary data?

current structure is as below
```
"total": {
  "requests": 102,
  "exceptions": 1,
  "write_errors": 0,
  "read_errors": 0,
  "pid": 1
}
```

turned around might look like this
```
"summary": {
  "requests": {
     "total": 102
  },
  "exceptions": 1,
  "errors": {
    "write": {
       "total" : 0
    },
    "read": {
       "total" : 0
    },
  }
  "pid": 1
}
```

feels like to much overhead, but if it is considered as more readable I welcome the changes.
I appreciate if you can suggest better structure for summary data.

Reply:
In general my approach is going from the generic one to the more detailed. So in the above case I think a user might first search for an `error` and then he can decide if writes or reads. I would assume he does not look at the document and think "I need a total" and then check what is inside total. But I'm pretty sure it could be argued the other way around :-)